client/src/components/Contact.js

```diff
@@ -1,14 +1,40 @@
-import React, { Component, Fragment } from 'react';
+import React, { Fragment } from 'react';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import * as S from './Contact.style';
 import CHeader from './CHeader';
+import CFooter from './CFooter';
 
-class Contact extends Component {
-  render() {
-    return (
-      <Fragment>
-        <CHeader menu />
-      </Fragment>
-    );
-  }
-}
+const Contact = () => {
+  return (
+    <Fragment>
+      <CHeader menu />
+      <S.Main>
+        <S.H1>Contact</S.H1>
+        <S.ContactWrapper>
+          <S.H3>
+            <u>
+              business
+              <br />
+              launchpad
+            </u>
+          </S.H3>
+          <S.P>
+            Trident Business Centre <br /> 89 Bickersteth Rd,
```

Comment:
generally best not to use ```<br />``` for styling!
Instead you can put each new line in its own block element.
That means each line would be wrapped in

```<S.P>New line here</S.P>```
and you would remove any excess spacing in your styled component